automatic/powertoys/update.ps1

```diff
@@ -5,19 +5,14 @@ $releases = 'https://api.github.com/repos/microsoft/PowerToys/releases/latest'
 function global:au_SearchReplace {
   return @{
     'tools\chocolateyInstall.ps1' = @{
-      "(^[$]fileName\s*=\s*`"[$]toolsDir\\).*" = "`${1}$($Latest.FileName64)`""
       "(^[$]version\s*=\s*)`".*`""             = "`${1}`"$($Latest.RemoteVersion)`""
-    }
-    ".\legal\VERIFICATION.txt"    = @{
-      "(?i)(listed on\s*)\<.*\>" = "`${1}<$releases>"
-      "(?i)(64-Bit.+)\<.*\>"     = "`${1}<$($Latest.URL64)>"
-      "(?i)(checksum type:).*"   = "`${1} $($Latest.ChecksumType64)"
-      "(?i)(checksum64:).*"      = "`${1} $($Latest.Checksum64)"
+      "(^\s*[$]url64\s*=\s*)('.*')"                      = "`$1'$($Latest.URL64)'"
+      "(^\s*[$]checksum64\s*=\s*)('.*')"                 = "`$1'$($Latest.Checksum64)'"
     }
   }
 }
 
-function global:au_BeforeUpdate { Get-RemoteFiles -NoSuffix -Purge }
+function global:au_BeforeUpdate { $Latest.Checksum64 = Get-RemoteChecksum $Latest.URL64 }
```

Comment:
Done.

I had used this method to get the checksum as it is more portable, since the AU built in check summing feature does not work on non-windows systems, but it is no biggy either way.